examples/pytorch/rrn/rrn.py

```diff
@@ -0,0 +1,52 @@
+import torch
```

Comment:
Please provide header comments:
```suggestion
"""Recurrent Relational Network(RRN) module

References:
- Recurrent Relational Networks
- Paper: https://arxiv.org/abs/1711.08028
- Code: https://github.com/rasmusbergpalm/recurrent-relational-networks
"""
import torch
```